release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1026,7 +1028,12 @@ In the table below, features are marked with the following statuses:
 |OpenShift Pipelines
 |TP
 |TP
-|
+|TP
+
+|OpenShift GitOps
+|-
+|-
+|TP
```

Comment:
```GA```. As we are not sync with OpenShift release, our GA will come after 4.7 is released.

Reply:
After discussion over slack with @Preeticp got to know that until the GA is out we can not mark it as GA. Lets go this field as TP.